src/main/java/duke/util/DukeDB.java

```diff
@@ -0,0 +1,171 @@
+package duke.util;
+
+import duke.Duke;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class DukeDB {
```

Comment:
Consider adding javadocs for this class and its methods! Because they are not very straightforward